src/js/visual/configurator-csr.js

```diff
@@ -1309,6 +1309,9 @@ ripe.ConfiguratorCsr.prototype._unpackSceneOptions = function(options) {
     const cameraOptions = {};
     const zoomOptions = {};
 
+    // unpacks root options
+    const environment = options.environment;
+

```

Comment:
move this closer to the place where it's going to be used
or if you're ordering in the same order as the return then move the zoom block to be the last one

Reply:
This is a root value, it's following the same order as the other unpack methods